Core/System.cpp

```diff
@@ -593,6 +593,8 @@ std::string GetSysDirectory(PSPDirectories directoryType) {
 			return g_Config.appCacheDirectory;
 		}
 		return g_Config.memStickDirectory + "PSP/SYSTEM/CACHE/";
+	case DIRECTORY_VIDEO_DUMP:
+		return g_Config.memStickDirectory + "PSP/FRAMEDUMP/";
```

Comment:
I guess `PSP/VIDEO/` is probably a more PSP-ish place to put this?

-[Unknown]